src/NodaTime.Web/Markdown/root/versions.md

```diff
@@ -708,3 +711,16 @@ Bug fixes:
 ## 1.0.0-beta1, released 2012-04-12 with tzdb 2012c
 
 - Initial beta release
+
+# <a name="tzdb-updates"></a>Update policy for time zone data
+
+New time zone data is released by IANA as required. The latest data
+is always available via the Noda Time web site as described in the
+[user guide](/userguide/tzdb). Additionally, NuGet packages are
+updated with a patch release to include the latest data.
+
+The latest minor release for each major release is always updated.
```

Comment:
That's just a complicated way of saying "the latest release", isn't it? :)

Reply:
Nope - "1.4" isn't the latest release, but it's the latest minor release of a major release.

Reply:
I'm apparently unable to read. Thanks for the clarification :)